server_addon/applications/server/settings.py

```diff
@@ -144,7 +144,17 @@ def validate_unique_name(cls, value):
 
 
 class ApplicationsSettings(BaseSettingsModel):
-    """Applications settings"""
+    """Applications settings
+
+    Define applications are available to your studio, locations of their
+    executables, and their additional environments.
```

Comment:
```suggestion
    Define applications available to your studio, locations of their
    executables, and their additional environments.
```
---
OR alternatives:

```suggestion
    Define applications that are available to your studio, locations of their
    executables, and their additional environments.
```
OR
```suggestion
    Define applications which are available to your studio, locations of their
    executables, and their additional environments.
```

Reply:
Maybe also `additional environments` is better written as `additional environment variables` for full clarity?